src/main/java/com/direwolf20/laserio/util/VectorHelper.java

```diff
@@ -12,19 +11,16 @@
  * this means that we can use COLLIDER so it traces through non-collidable objects
  */
 public class VectorHelper {
-    public static BlockHitResult getLookingAt(Player player, ItemStack tool, int range) {
+
+    public static final int RANGE = 10; // How far away you can click on blocks from
```

Comment:
perhaps better to move to a settings/config class or better: configuration file?